hal/pinmap.h

```diff
@@ -22,6 +22,7 @@
 
 #include "PinNames.h"
 #include <stdbool.h>
+#include <stdint.h>
```

Comment:
please send this as separate pull request

Reply:
In my latest version this change is not existing any more. It should be good practice to include <stdint.h> in pinmap.h because the standard integer types are used in this file. Currently pinmap.h is relying on PinNames.h to include this file. Since I am now also including stdint.h in my PinNames.h, the above change is not necessary any more.